config/rollup-full-build.js

```diff
@@ -1,22 +1,24 @@
 import commonjs from '@rollup/plugin-commonjs';
-import esbuild from 'rollup-plugin-esbuild';
```

Comment:
Curious if you saw an improvement using terser over esbuild.  If we don’t use this plugin, it can be removed from package.json.

Reply:
Yes, the build gets 11k smaller with terser. Removing the esbuild plugin now.